robottelo/cli/factory.py

```diff
@@ -92,6 +92,12 @@ def create_object(cli_object, options, values):
     :return: A dictionary representing the newly created resource.
 
     """
+    if values:
+        diff = set(values.keys()).difference(set(options.keys()))
+        logger.debug(
```

Comment:
This way you'll always receive the warning, each call. You should only rise a warning if diff is not empty.